google-youtube.html

```diff
@@ -647,6 +747,9 @@
       this.currenttime = Math.round(this._player.getCurrentTime());
       this.currenttimeformatted = this._toHHMMSS(this.currenttime);
       this.fractionloaded = this._player.getVideoLoadedFraction();
+      this.volume = this._player.getVolume();
+      this.playbackrate = this._player.getPlaybackRate();
```

Comment:
I think you can remove these two lines now that you're setting the values inside `onPlaybackQualityChange`/`onPlaybackRateChange`.